src/Language/PureScript/Ide/Reexports.hs

```diff
@@ -1,6 +1,22 @@
+-----------------------------------------------------------------------------
+--
+-- Module      : Language.PureScript.Ide.Reexports
+-- Description : Resolves reexports for psc-ide
+-- Copyright   : Christoph Hegemann 2016
+--               Brian Sermons 2016
```

Comment:
We can ping @bsermons and ask him to open a PR against my  kRITZCREEK:import-helper-commands branch